src/components/CertificateDropZone/Views/index.ts

```diff
@@ -0,0 +1,3 @@
+export const unverifiedMessage = "What Should I Do";
+export const docNotValidMessage = "This document is not valid";
+export const tryAnotherMessage = "Try another document";
```

Comment:
```suggestion
export enum VerifyMessage {
  UNVERIFIED = "What Should I Do",
  INVALID = "This document is not valid",
  RETRY = "Try another document"
}
```

this way could make use of the autocomplete to "see" all states

Reply:
hmm, i could do that, but this is mainly to prep for future use case of moving towards localisation.

actually would it be smart to just have a file with all our texts in them and organised properly?

something like `messages/en.ts`

```ts
export default text = {
  [<component>-<message-description>]: "message text",
  "verify-unverify": "What Should I Do",
  "verify-invalid": "This document is not valid",
}
....
```